test/integration_test.go

```diff
@@ -79,7 +79,7 @@ func testStackWithOptions(
 ) (*jobcreator.RunJobResults, error) {
 	// give the solver server a chance to boot before we get all the websockets
 	// up and trying to connect to it
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(20000 * time.Millisecond)
```

Comment:
this is a big jump. what in this PR calls for the increase (when tests seem to be passing on most other PRs without)?

Reply:
Yes, I agree with you and i'm not quite sure what the issue since theoretically this should start up faster with the reduced network call.  I experimented with a bump to 200, 10000 and 20000 milliseconds and only the latter ended up giving me consistent results.